docs/api-guide/testing.md

```diff
@@ -102,6 +102,20 @@ This means that setting attributes directly on the request object may not always
     request.user = user
     response = view(request)
 
+If you want to test a request involving the REST framework’s 'Request' object, you’ll need to manually transform it first:
+
+    class DummyView(APIView):
+        ...
+
+    factory = APIRequestFactory()
+    request = factory.get('/', {'demo': 'test'})
+    DRF_request = DummyView().initialize_request(request)
+    assert DRF_request.query_params == {'demo': ['test']}
+
+    request = factory.post('/', {'example': 'test'})
+    DRF_request = DummyView().initialize_request(request)
+    assert DRF_request.data.get('example') == 'test'
```

Comment:
It seems to work by using the `APIView` directly, let's not get distracted with a subclass, shall we? Also, let's try to follow pep8 (lowercase snake case for variables):

```suggestion
    factory = APIRequestFactory()
    django_request = factory.get('/', {'demo': 'test'})
    request = APIView().initialize_request(django_request)
    assert request.query_params == {'demo': ['test']}

    django_request = factory.post('/', {'example': 'test'})
    request = APIView().initialize_request(django_request)
    assert request.data.get('example') == 'test'
```

Same simplification for the test